build.go

```diff
@@ -0,0 +1,44 @@
+package rake
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/cloudfoundry/packit/scribe"
+	"github.com/paketo-buildpacks/packit"
+)
+
+//go:generate faux --interface Parser --output fakes/parser.go
+type Parser interface {
+	Parse(path string) (hasRakeGem bool, err error)
+}
+
+func Build(gemfileParser Parser, logger scribe.Logger) packit.BuildFunc {
+	return func(context packit.BuildContext) (packit.BuildResult, error) {
+		logger.Title("%s %s", context.BuildpackInfo.Name, context.BuildpackInfo.Version)
+
+		hasRakeGem, err := gemfileParser.Parse(filepath.Join(context.WorkingDir, "Gemfile"))
+		if err != nil {
+			return packit.BuildResult{}, fmt.Errorf("failed to parse Gemfile: %w", err)
+		}
+
+		var command string
+		if hasRakeGem {
+			command = "bundle exec rake"
+		} else {
+			command = "rake"
+		}
+
+		logger.Process("Writing rake command")
+		logger.Subprocess(command)
```

Comment:
Can we make this look like https://github.com/paketo-buildpacks/go-build/blob/7c6818b3f4e12ac88c0b2c8ca72d6182309e5b25/build.go#L107-L108? I would like us to move towards a generic form for all buildpacks that set a launch process with an output that allows for multiple process types to be shown.